forensics/Copy-VmDigitalEvidence.ps1

```diff
@@ -62,6 +68,7 @@ if ($bios) {
     # The following doc shows a possible way to create a persistent mount point:
     # https://docs.microsoft.com/azure/storage/files/storage-how-to-use-files-linux#mount-the-azure-file-share-on-demand-with-mount
 
+    # put here your code to map the file share. Note that the file share needs to be mapped by this script, otherwise the runbook worker'll miss the right access to the share.

```

Comment:
```suggestion
    # Put your code here to map the file share. Note that the file share needs to be mapped by this script, otherwise the runbook worker will not function due to missing access to the share.
```